charts/workflow/values.yaml

```diff
@@ -122,13 +122,12 @@ database:
     host: "database host"
     port: "database port"
 
-logger:
-  redis:
-    # Configure the following ONLY if using an off-cluster Redis instance for logger
-    db: "0"
-    host: "redis host"
-    port: "redis port"
-    password: "redis password" # "" == no password
+redis:
+  # Configure the following ONLY if using an off-cluster Redis instance for logger
+  db: "0"
+  host: "redis host"
+  port: "redis port"
+  password: "redis password" # "" == no password
```

Comment:
what version are you running? We haven't released a chart with this fix yet so it shouldn't occur with v2.9.0 or earlier. :)